project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -564,6 +564,26 @@ function c4m_message_format_event_user_full_name(Message $message) {
 }
 
 /**
+ * Returns a link to event user profile.
+ *
+ * @param Message $message
+ *   The Message entity.
+ *
+ * @return string
+ *   Returns a link to new member public profile.
+ */
+function c4m_message_format_event_user_url(Message $message) {
+  $wrapper = entity_metadata_wrapper('message', $message);
+  $uid = $wrapper->field_related_user->value(array('identifier' => TRUE));
```

Comment:
let's use one that will have the same purpose for all, trying to be consistent and explicit